spikeinterface/sorters/tests/test_runsorter.py

```diff
@@ -18,22 +19,46 @@ def test_run_sorter_local():
     print(sorting)
 
 
-@pytest.mark.skip('Docker test no run with pytest : do it manually')
+@pytest.mark.skip("Docker tests don't run with pytest : test it manually")
 def test_run_sorter_docker():
-    local_path = download_dataset(remote_path='mearec/mearec_test_10s.h5')
-    recording, sorting_true = read_mearec(local_path)
+    mearec_filename = download_dataset(remote_path='mearec/mearec_test_10s.h5')
+
+    base_folder = Path('/data_local/DataSpikeSorting/mearec')
+    mearec_filename = base_folder / 'recordings_collision_10cells_Neuronexus-32_180s.h5'
```

Comment:
Shouldn't this be the file that you downloaded in the lines above?

Reply:
downlaod with datalad is a pain for now because of version of git-annex.
We need a big refactor about this.
We keep this local test with local test for now.
If you want to test you can use you own files.